### PR TITLE
depend on the nodejs role

### PR DESCRIPTION
**Pull request by kayiwa**
the tasks are not needed. Make dss a meta role

### CODE REVIEW

File: roles/pulibrary.nodejs/tasks/main.yml [hunk 1]
```diff
@@ -1,28 +1,14 @@
...
-- name: Get upstream APT GPG key
-  apt_key:
-    id: "{{ nodejs__upstream_key_id }}"
-    keyserver: "hkp://pool.sks-keyservers.net"
-    state: present
-  register: pool_servers
-  ignore_errors: true
+- name: Download node apt update script
+  get_url:
+    url: https://deb.nodesource.com/setup_10.x
```
Comment (jrgriffiniii) [inline roles/pulibrary.nodejs/tasks/main.yml]: 👍 , but is this entirely secure?

File: roles/pulibrary.nodejs/tasks/main.yml [hunk 2]
```diff
@@ -41,28 +27,44 @@
...
+- name: Uninstall Node.js
+  when: node_version.stdout is search( node_major )
+  apt:
+    name: "nodejs"
+    state: absent
+  changed_when: false
+
 - name: Install node packages
   apt:
-    name: ["nodejs", "yarn"]
```
Comment (jrgriffiniii) [inline roles/pulibrary.nodejs/tasks/main.yml]: 👍

File: roles/pulibrary.nodejs/tasks/main.yml (hunk 1 above)
Comment (kayiwa) [reply roles/pulibrary.nodejs/tasks/main.yml]: how do you mean here? @jrgriffiniii you want this checksummed?  other?

Comment (jrgriffiniii) [reply roles/pulibrary.nodejs/tasks/main.yml]: Sorry, I meant that this is a cURL to a shell script hosted on the Web, it's not a blocking concern from me, please disregard.

Comment (kayiwa) [reply roles/pulibrary.nodejs/tasks/main.yml]: I am using the `apt-get` infrastructure now. I'd borrow from your other PR to use the `curl` -> `command`

**Pull request closed without merging**